python-threatexchange/threatexchange/exchanges/clients/techagainstterrorism/api.py

```diff
@@ -121,7 +126,7 @@ def get_auth_token(self, username: str, password: str) -> t.Optional[str]:
 
         if not isinstance(username, str) or not isinstance(password, str):
```

Comment:
nit: This is what typing is for, you don't need to check it at runtime. You might be missing a class-level annotation, but most modern typecheckers intuit from `__init__`